### ISSUE TITLE
Cannot use volumes with TPU v6e

### ISSUE THREAD

**Issue by r4victor** [dstackai/dstack#2145]
#2144 added support for volumes with all TPU versions except TPU v6e. Attaching volumes to TPU v6e fails on the GCP side. Here's the GCP issue: https://issuetracker.google.com/issues/385924880

**Comment by peterschmidt85** [comment]
The issue seems to be fixed. Should it work on our side or we need to do something?

**Comment by r4victor** [comment]
TPU v6e doesn't support Persistent disks. Requires https://github.com/dstackai/dstack/issues/2588 for volumes to work with TPU v6e.

**Comment by peterschmidt85** [comment]
@r4victor BTW what about v7?

**Comment by r4victor** [comment]
After #2685, we issue an error if persistent disk is not supported by the instance type. This issue is superseded by #2588
